datahub-web-react/src/app/previewV2/DefaultPreviewCard.tsx

```diff
@@ -218,6 +229,15 @@ export default function DefaultPreviewCard({
 }: Props) {
     const entityRegistry = useEntityRegistryV2();
     const supportedCapabilities = entityRegistry.getSupportedEntityCapabilities(entityType);
+    const { config } = useAppConfig();
+
+    const shouldShowRedesignedDescription = config.searchCardConfig.showDescription;
+    const shouldShowDescriptionsForSearch =
+        previewType === PreviewType.SEARCH && config.searchCardConfig.showDescription;
+    const shouldShowDescription =
+        previewType === PreviewType.HOVER_CARD ||
+        ENTITY_TYPES_WITH_DESCRIPTION_PREVIEW.has(entityType) ||
+        shouldShowDescriptionsForSearch;
```

Comment:
yeah so this logic would have to change slightly with that change i mention in my previous comment